docs/_data/meetups/qrm-usEast.yaml

```diff
@@ -0,0 +1,12 @@
+region: Quorum
+country: USA
+state: East Coast
+city:
+website: https://www.meetup.com/virtual-write-the-docs-east-coast-quorum/
+meetup: Write-The-Docs-East-Coast
+twitter:
+organizers:
+  - name: Alyssa Whipple Rock
+    link: 
+  - name: Rose
```

Comment:
Is the intent here to link to my professional website? If so, my website is: https://alyssarock.pro/

Reply:
Twitter would work for me https://twitter.com/ZelWms

Alternatively, we could reference the other meetups organizers work with.

Reply:
Sounds good. Here's the list of participating organizers and chapters:

The following U.S. East Coast and Central meetups are currently participating in Quorum:

Atlanta, GA - Alan Bowman
Austin, TX - Shrie Spangler
Boston, MA - Rose, are you the organizer here?
Detroit, MI/Windsor, CAN - Ryan Macklin and Ashley Newton
Durham, NC - Susan Tacker
Florida - Barry Byron
Ottawa/Montreal, CAN - Shay Perez
Philadelphia, PA - Leslie Salazar

The following U.S. West Coast and Mountain meetups are currently participating in Quorum:

Bay Area, CA - Liz Harris, Clarence Cromwell, Maggie Fero
Los Angeles, CA - Andrea Kao
Portland, OR - Mike Jang
Salt Lake City, UT - Alyssa Rock

If you want links to the chapters and each of these individuals on Meetup, I recommend using the links from this page that I discovered recently: https://www.writethedocs.org/meetups/index.html

Reply:
The listing for the Quorum meetups, once I figure out the build, will be on the same page as the other meetups. So I think, for now, I'll "just" include a list to the other meetups (Quorum members).

Reply:
Here's a screenshot of my WIP build:

![QuroumMeetupChange](https://user-images.githubusercontent.com/3287976/107864230-d4940500-6e0e-11eb-9a89-5e63ce0aea6b.png)

Adding everyone's organizer name, in the current YAML format, requires some detailed knowledge of rST and Sphinx that I don't have. The patterns don't look hard, but I've already spent a bunch of time on this admitted hack.

@barbaricyawps wdyt? @plaindocs I'd appreciate your comments as well, given the hackiness of what I'm doing.

I recognize this leaves open some other needed changes, such as adding Raleigh-Durham to the Meetup list.

Reply:
I'm totally fine with leaving names off for now if it's easier. Thanks for doing this, @mjang !

Reply:
FYI, I noticed most of the current organizers are listed here if you just want to link to this page: https://www.writethedocs.org/meetups/index.html

Reply:
`Hi @barbaricyawps , this PR adds info to the bottom of that page.`